modules/efk-logging-about-components.adoc

```diff
@@ -5,13 +5,13 @@
 [id="efk-logging-about-components_{context}"]
 = About cluster logging components 
 
-There are currently 4 different types of cluster logging components:
+There are currently 5 different types of cluster logging components:
 
 * logStore - This is where the logs will be stored. The current implementation is Elasticsearch.
-* collection - This is the component that collects logs from the node, formats them, and stores them in the logStore. The current implementation is Fluentd.
+* collection - This is the component that collects logs from the node, formats them, and stores them in the logStore. The current implementation is Fluentd, default, or Rsyslog.
```

Comment:
Seems we don't have an default option here.

Reply:
@QiaolingTang Thank you. I will correct this throughout. There is no default. The user needs to specify the collector in the CR.